<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesUI</class>
 <widget class="QDialog" name="preferencesUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>463</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/ball.ico</normaloff>:/res/ball.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="18" column="0" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Plotting</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data Loading</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="prefsPlotting">
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>331</width>
             <height>171</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Default Units On Plotting</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="6" column="1">
               <widget class="QComboBox" name="cbPlotIAbsSquared"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Inverse length</string>
                </property>
               </widget>
              </item>
              <item row="5" column="3">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Arbitrary</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>SESANS</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QComboBox" name="cbPlotIAbs"/>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="cbPlotQLength"/>
              </item>
              <item row="6" column="3">
               <widget class="QComboBox" name="cbPlotIArbitrary"/>
              </item>
              <item row="3" column="0">
               <widget class="QComboBox" name="cbPlotQInvLength"/>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>(Abs. Units)^2</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QComboBox" name="cbPlotISesans"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Absolute Units</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Length</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2" colspan="2">
               <widget class="QCheckBox" name="checkBoxPlotQAsLoaded">
                <property name="text">
                 <string>Plot Q As Loaded</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="2" colspan="2">
               <widget class="QCheckBox" name="checkBoxPlotIAsLoaded">
                <property name="text">
                 <string>Plot I As Loaded</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Intensity Units</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Q Units</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="prefsDataLoading">
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>331</width>
             <height>171</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The units selected in this section will be attached to the X and Y values of a data set when the loaded data set has no known units. Units found within a data file take precedence over the values selected here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Default Units On Loading</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Q Units</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QComboBox" name="cbLoadIUnitSelector"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_18">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Intensity Units</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QComboBox" name="cbLoadIUnitType">
                <item>
                 <property name="text">
                  <string>Absolute Units</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>(Absolute Units)^2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>SESANS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Arbitrary</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Units</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Unit Type</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Unit Type</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QComboBox" name="cbLoadQUnitType">
                <item>
                 <property name="text">
                  <string>Length</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Inverse Length</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Units</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="cbLoadQUnitSelector"/>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="checkBoxLoadQOverride">
                <property name="text">
                 <string>Override File Q Units</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="checkBoxLoadIOverride">
                <property name="text">
                 <string>Override File I Units</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="22" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferencesUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferencesUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
